src/rocm_docs/data/projects.yaml

```
@@ -82,6 +83,7 @@ projects:
   rocprofiler-sdk:
     target: https://rocm.docs.amd.com/projects/rocprofiler-sdk/en/${version}
     development_branch: amd-mainline
+  rocpydecode: https://rocm.docs.amd.com/projects/rocppydecode/en/${version}
```

```suggestion
  rocpydecode: https://rocm.docs.amd.com/projects/rocPyDecode/en/${version}
```

links are case sensitive

---

Ah, thx Sam, wasn't aware.